pkg/configuration/configuration.go

```diff
@@ -166,12 +184,41 @@ func New(configFilePath string) (*Registry, error) {
 	return c, nil
 }
 
+func getRegistrationEnvVarKey(key string) string {
+	envKey := strings.ToUpper(strings.ReplaceAll(key, ".", "_"))
+	return EnvPrefix + "_" + envKey
+}
+
+func (c *Config) PrintConfig() {
+	logWithValuesHost := log.WithValues(
```

Comment:
Is it possible to list all available keys in viper, so we don't have to hardcode all the keys in `PrintConfig()`?

Reply:
yeah it is, so it will print this way using the keys: 
```
{"level":"info","ts":1598366024.6639636,"logger":"configuration","msg":"Registration service configuration variablesz:","log.level":"info","http.idle_timeout":15,"environment":"dev","verification.enabled":true,"verification.daily_limit":5,"http.compress":true,"verification.message_template":"Your verification code for Red Hat Developer Sandbox is: %s","graceful_timeout":15,"auth_client.config.content_type":"application/json; charset=utf-8","auth_client.library_url":"https://sso.prod-preview.openshift.io/auth/js/keycloak.js","namespace":"tkurian-host-operator","http.write_timeout":15,"log.json":false,"auth_client.public_keys_url":"https://sso.prod-preview.openshift.io/auth/realms/toolchain-public/protocol/openid-connect/certs","verification.attempts_allowed":3,"http.read_timeout":15,"http.address":"0.0.0.0:8080","auth_client.config.raw":"{\n\t  \"realm\": \"toolchain-public\",\n\t  \"auth-server-url\": \"https://sso.prod-preview.openshift.io/auth\",\n\t  \"ssl-required\": \"none\",\n\t  \"resource\": \"crt\",\n\t  \"clientId\": \"crt\",\n\t  \"public-client\": true\n\t}"}

```

updated here: 1e5dc999120b13ce69d2421334dd1d7600df16d8